coverage/.resultset.json

```diff
@@ -0,0 +1,268 @@
+        null,
+        0,
+        0,
+        null,
+        null,
+        null,
+        1,
+        1,
+        null,
+        null
+      ],
+      "/Users/robinsonalex/Turing/projects/opportunity_at_work/app/controllers/static_pages_controller.rb": [
+        1,
+        1,
+        null,
+        null,
+        1,
+        null,
+        null,
+        1,
+        1,
+        null,
+        null,
+        1,
+        1,
+        null,
+        null
+      ],
+      "/Users/robinsonalex/Turing/projects/opportunity_at_work/app/controllers/admin/tools_resources_controller.rb": [
+        1,
+        1,
+        null,
+        null
+      ],
+      "/Users/robinsonalex/Turing/projects/opportunity_at_work/app/controllers/companies_controller.rb": [
+        1,
+        1,
+        3,
+        null,
+        null,
+        1,
+        3,
+        3,
+        2,
+        2,
+        null,
+        1,
+        1,
+        null,
+        null,
+        null,
+        1,
+        null,
+        1,
+        3,
+        null,
+        null,
+        null,
+        null,
+        null,
+        null,
+        null,
+        null,
+        null
+      ],
+      "/Users/robinsonalex/Turing/projects/opportunity_at_work/app/controllers/pdfs_controller.rb": [
+        1,
+        1,
+        1,
+        1,
+        1,
+        null,
+        null,
+        null
+      ],
+      "/Users/robinsonalex/Turing/projects/opportunity_at_work/app/models/company.rb": [
+        1,
+        1,
+        null,
+        1,
+        null,
+        1,
+        1,
+        1,
+        null,
+        1,
+        null,
+        1,
+        1,
+        1,
+        1,
+        null,
+        1,
+        null,
+        46,
+        null,
+        1,
+        30,
+        null,
+        null,
+        1,
+        0,
+        null,
+        null,
+        1,
+        null
+      ],
+      "/Users/robinsonalex/Turing/projects/opportunity_at_work/app/models/note.rb": [
+        1,
+        1,
+        null
+      ],
+      "/Users/robinsonalex/Turing/projects/opportunity_at_work/app/models/pdf.rb": [
+        1,
+        1,
+        null,
+        null,
+        null,
+        1,
+        null,
+        null,
+        null,
+        null,
+        1,
+        null,
+        1,
+        1,
+        null,
+        null,
+        1,
+        13,
+        null,
+        null,
+        1,
+        5,
+        null,
+        null
+      ],
+      "/Users/robinsonalex/Turing/projects/opportunity_at_work/app/controllers/admin/dashboard_controller.rb": [
+        1,
+        1,
+        null,
+        1,
+        null,
+        null,
+        null
+      ],
+      "/Users/robinsonalex/Turing/projects/opportunity_at_work/app/controllers/admin/notes_controller.rb": [
+        1,
+        1,
+        1,
+        1,
+        1,
+        null,
+        null,
+        1,
+        null,
+        1,
+        1,
+        null,
+        null
+      ]
+    },
+    "timestamp": 1430264034
+  }
+}
```

Comment:
I have this in my gitignore:
# Ignore simplecov coverage
coverage

Is that sufficient?